### PR TITLE
feat: add users meta base models

### LINKED ISSUE #180: Add users meta base models
**What feature do you want to improve?**
Add `medic-users-meta` models from [cht-couch2pg](https://github.com/medic/cht-couch2pg/tree/main/libs/medic-users-meta/migrations)

**Describe the improvement you'd like**
We should make existing user base models available as a part of cht-sync adn cht-pipleine as they were required and being used everywhere. 

**Describe alternatives you've considered**
Continue using cht-couch2pg.

**Additional context**

### PR DESCRIPTION
**Pull request by njuguna-n**
# Description

Add telemetry and feedback base models

closes #180 

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

**Review by witash — APPROVED** [review]
LGTM

**Comment by jdndiaye** [comment]
Hi @njuguna-n Are you planning to add any more telemetry to this model? Based on this [list](https://docs.communityhealthtoolkit.org/building/guides/performance/telemetry/), there's still more to add. cc @1yuv

**Comment by njuguna-n** [comment]
Hello @jdndiaye, this PR is meant to provide base models similar to the [ones](https://docs.communityhealthtoolkit.org/hosting/analytics/building-dbt-models/#base-models) from the main database and not extract all available telemetry data. Are there metrics common across deployments that we should here?

**Comment by jdndiaye** [comment]
> Are there metrics common across deployments that we should here?
 I know that some of them depend on the version deployed. But we can just limit ourselves to the indicators listed [here](https://docs.google.com/spreadsheets/d/1d9nI5MnPQvgAOZ3Kw6b8mvLagVIdvpS8OrVbPZYRLMw/edit?gid=1994745905#gid=1994745905).

File: models/users/telemetry.sql [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+        (doc#>>'{metadata,month}')::int     --> Month is between 1 - 12
+    END,
+    CASE                                    --> day of the month, else 1
+      WHEN
+        (doc#>>'{metadata,day}') IS NOT NULL
+      THEN
+        doc#>>'{metadata,day}'
+      ELSE
+        '1970-01-01'
+    END
```
Comment (1yuv) [inline models/users/telemetry.sql:38]:
```suggestion
      ELSE
        '1'
    END
```

This is not correct, if day dosn't exist, we should set it to 1.

**Review by 1yuv — CHANGES_REQUESTED** [review]

**Pull request merged**